### PR TITLE
splashy

### PR DESCRIPTION
**Pull request by johngruen**
good work john.

### CODE REVIEW

**Comment by johngruen** [comment]
thanks john!

File: app/views/home.html [hunk 1]
```diff
@@ -0,0 +1,31 @@
+.full.fancy-medium
```
Comment (pdehaan) [inline app/views/home.html]: Should this page be "home.html" or "home.jade"?

File: gulpfile.js [hunk 2]
```diff
@@ -66,12 +68,22 @@ gulp.task('bower', function() {
     .pipe(notify({ message: 'Vendor libs ready' }));
 });
 
+gulp.task('bower:tabzilla', function() {
+  return gulp.src('components/mozilla-tabzilla/media/**')
+    .pipe(gulp.dest('public/vendor/mozilla-tabzilla/css/media'))
+});
+
+gulp.task('static', function() {
+  gulp
```
Comment (pdehaan) [inline gulpfile.js]: What's this?

Comment (johngruen) [inline gulpfile.js]: @pdehaan it's the new tabzilla bower package being kinda questionable

Comment (johngruen) [inline gulpfile.js]: oh, i mean, that's me forgetting to lint stuff :)

File: src/styles/_mixins.scss [hunk 3]
```diff
@@ -34,3 +36,11 @@ $breakpoints: (
   width: 768px;
   margin: 0 auto;
 }
+
+@mixin clear-fix() {
+  &:after {
+  clear: both;
+  content: "";
+  display: table;
```
Comment (pdehaan) [inline src/styles/_mixins.scss]: nit: indenting is "off" here.

File: src/styles/_variables.scss [hunk 4]
```diff
@@ -1,5 +1,14 @@
+$url-blue: #faf;
```
Comment (pdehaan) [inline src/styles/_variables.scss]: Not sure if I'm colorblind, but is `$url-blue` actually pink? http://www.colorhexa.com/ffaaff

**Comment by pdehaan** [comment]
I think this PR broke an embedded image in the [README.md](https://github.com/mozilla/idea-town#idea-town) file:

mozilla_idea-town

**Pull request merged**